src/scripts/main.js

```diff
@@ -1 +1,53 @@
 'use strict';
+
+function insertMessage(messageClass, message) {
+  document.body.insertAdjacentHTML('beforeend',
+    `<div data-qa="notification" class = ${messageClass}> ${message}</div>`
+  );
+};
+
+const promise1 = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => resolve());
```

Comment:
```suggestion
  document.addEventListener('click', () => resolve('First promise was resolved'));
```
just want to make sure you know it:
you can pass this message here and then get it as an argument of the callback function you passed in `then` method.
The same with error and `catch`